hack/update-deps.sh

```diff
@@ -29,13 +29,13 @@ FLOATING_DEPS=(
 )
 
 REPLACE_DEPS=(
-  "knative.dev/eventing-kafka-broker=github.com/openshift-knative/eventing-kafka-broker@${EVENTING_KAFKA_BROKER_VERSION}"
-  "knative.dev/eventing=github.com/openshift-knative/eventing@${EVENTING_VERSION}"
-  "knative.dev/serving=github.com/openshift-knative/serving@${SERVING_VERSION}"
   "knative.dev/pkg=knative.dev/pkg@${KN_VERSION}"
   "knative.dev/hack=knative.dev/hack@${KN_VERSION}"
   "knative.dev/networking=knative.dev/networking@${KN_VERSION}"
   "knative.dev/reconciler-test=knative.dev/reconciler-test@${KN_VERSION}"
+  "knative.dev/eventing-kafka-broker=github.com/openshift-knative/eventing-kafka-broker@${EVENTING_KAFKA_BROKER_VERSION}"
```

Comment:
reordered to make sure the script runs. Eventing deps need new version of upstream deps first